motoko/canister_logs/Makefile

```diff
@@ -36,19 +36,21 @@ test: install
 		| grep 'print via non-replicated query' && echo 'PASS'
 
 	# Test trapped call is recorded in logs. 
-	# Ignore failed call output (so the test continues) and check the logs to contain the message.
-	-dfx canister call CanisterLogs trap 'trap via update'
+	# Ignore failed dfx command output (so the test continues) and check the logs to contain the message.
+	- dfx canister call CanisterLogs trap 'trap via update'
```

Comment:
For my understanding: where does this `- dfx` syntax come from? I cannot use it in a bash shell.